pkg/google/gke/pricing_map.go

```diff
@@ -124,14 +124,6 @@ func NewPricingMap(ctx context.Context, billingService *billingv1.CloudCatalogCl
 	return pm, nil
 }
 
-// NewComputePricingMap returns a new PricingMap in a way that can be used afterwards.
-func NewComputePricingMap() *PricingMap {
-	return &PricingMap{
-		Compute: map[string]*FamilyPricing{},
-		Storage: map[string]*StoragePricing{},
-	}
-}
-
```

Comment:
This was no longer used after removing GCP compute module in https://github.com/grafana/cloudcost-exporter/pull/317